src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java

```diff
@@ -313,12 +313,12 @@ public int getDatabaseMinorVersion() throws SQLException {
 
 	@Override
 	public int getJDBCMajorVersion() throws SQLException {
-		return VersionUtil.getMajorDriverVersion();
+		return 4;
```

Comment:
Why are these values hardcoded?

Reply:
It is implemented in the second branch. I will merge it first, rebase this one and it will become not hard coded.

Reply:
I've rebased this branch on master and fixed this implementation: now it is not hard coded but the value is read from gradle.properties